test/wolt_modal_sheet_test.dart

```diff
@@ -263,4 +263,60 @@ void main() {
     expect(tester.getTopLeft(sheetMaterial), const Offset(0.0, 504.0));
     expect(tester.getTopRight(sheetMaterial), Offset(viewSize.width, 504.0));
   }, skip: true); // [Intended]: This is skipped due to a bug in the framework.
+
+  group('WoltModalSheet page getters', () {
+    testWidgets('getters return the correct page, page index and list of pages',
+        (tester) async {
+      const page1Id = 'page1';
+      const page2Id = 'page2';
+      final page1 =
+          WoltModalSheetPage(child: const Text('Page 1'), id: page1Id);
+      final page2 =
+          WoltModalSheetPage(child: const Text('Page 2'), id: page2Id);
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(body: Center(
+            child: Builder(builder: (context) {
+              return ElevatedButton(
+                onPressed: () {
+                  WoltModalSheet.show(
+                    context: context,
+                    pageListBuilder: (_) => [page1, page2],
+                  );
+                },
+                child: const Text('Open sheet'),
+              );
+            }),
+          )),
+        ),
+      );
```

Comment:
Please the new build widget method here with `pageListBuilder` overridden. It was added in https://github.com/woltapp/wolt_modal_sheet/pull/298

Like this 
```dart
    await tester.pumpWidget(buildSheetWithShow(
      pageListBuilder: (context) {
        return <WoltModalSheetPage>[]; 
    ));
```

Reply:
done 81a0ac30203256d0a4a252f7145a18cedef26263